src/components/TimePicker/TimePicker.story.jsx

```diff
@@ -1,3 +1,47 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean } from '@storybook/addon-knobs';
+import LinkTo from '@storybook/addon-links/react';
+
+import TimePicker, { TIMEGROUPS } from './TimePicker';
+
 export {
   default as TimePickerStory,
 } from 'carbon-components-react/lib/components/TimePicker/TimePicker-story';
+
+storiesOf('Watson IoT Experimental/TimePicker', module)
+  .add('Default', () => {
+    return (
+      <div>
+        Please refer to the{' '}
```

Comment:
We need to copy the stories from carbon and paste them in here. The library could be using a different version of Carbon than what is present in their storybook, the component signature could differ from what we offer.